mautrix_instagram/portal.py

```diff
@@ -1193,10 +1193,14 @@ async def _convert_instagram_media_share(
 
             if share_item.caption:
                 combined["com.beeper.raw_caption_text"] = share_item.caption.text
-                combined["com.beeper.instagram_item_username"] = share_item.caption.user.username
```

Comment:
Is this always covered by `share_item.user`?

Reply:
It looks that way to me, but we can do both (shouldn't hurt)